README.md

```diff
@@ -15,6 +15,11 @@ Docker command:
 
 	docker-compose up --build	
 
+or
```

Comment:
I might be a bit more verbose to help guide the next developer.  You could add a sub-section for local development, and indicate steps and requirements, like:

### LOCAL DEVELOPMENT

For local development where the web client and server are run directly off the host system some of the containers should be stopped to avoid port collisions and to reduce resources on containers that aren't required.

**Required Containers**

* primedb
* mailhog
* document-manager-backend
* prime-redis
 
To run the backend open `prime-dotnet-webapi` in VSCode and launch the API using the debugger (or F5).  For the web client run the web client locally type:

```
cd prime-angular-frontend
ng serve -o
```

I added this using markdown so if you want the markdown let me know, and I can pass it through Slack.

Reply:
That makes more sense, I love it. I got the markup from using edit on your comment :))